stable/yugaware/templates/service.yaml

```diff
@@ -99,6 +99,22 @@ spec:
           secret:
             secretName: {{ .Release.Name }}-yugaware-tls-cert
         {{- end }}
+      initContainers:
+        - image: busybox:1.33.0
```

Comment:
In the Yugabyte helm chart, we require busybox:1.31 (https://github.com/yugabyte/charts/blob/master/stable/yugabyte/templates/service.yaml#L392). Requiring two separate busybox images is a bit strange, especially for airgapped installations which will require the administrator to provide both. Should we use the same image in both places? Also, would we want to make this image + tag configurable via the values.yaml file like our other images? CC: @iSignal

Reply:
Also, just noticed https://github.com/yugabyte/charts/pull/54, which removes the other busybox image and replaces it with the yugabyte image. Maybe we would we want to use the yugaware image here, then?

Reply:
Agree that reusing an existing container would make it easier to deploy in offline environments where specific approved containers from an internal repo can be used.